manifests/istio-telemetry/mixer-telemetry/templates/deployment.yaml

```diff
@@ -219,9 +219,11 @@ spec:
         - name: sds-uds-path
           mountPath: /var/run/sds
           readOnly: true
+        {{- if eq .Values.global.jwtPolicy "third-party-jwt" }}
```

Comment:
Please also set condition for the "projected volume"

Reply:
Done.